front/src/App.tsx

```diff
@@ -75,27 +81,77 @@ function App() {
             <Route path=":hash" element={<ResolveHash />} />
           </Route>
           <Route path="boxes">
-            <Route index element={<Boxes />} />
+            <Route
+              index
+              element={<Protected component={<Boxes />} requiredAbp={["view_inventory"]} />}
```

Comment:
@aerinsol can you check from here to line 151 if the required action based permissions are correct?

Reply:
@HaGuesto this is "Manage Boxes" right? if so then it should be `manage_inventory` permission, `view_inventory` permission is only for Stock Overview